From: Jan Engelhardt <ej@inai.de>
Date: 2025-06-22 22:33:35.286193495 +0200

gcc-15 seems to come with fewer implicit cross-inclusions between headers.

[   41s] /home/abuild/rpmbuild/BUILD/polymake-4.13-build/polymake-4.13/include/core/polymake/internal/comparators.h:285:40: error: ‘uint64_t’ was not declared in this scope
[   41s]   285 | std::enable_if_t<sizeof(SizeT)==sizeof(uint64_t)> hash_combine(SizeT& h, SizeT k)

---
 include/core/polymake/internal/comparators.h |    1 +
 1 file changed, 1 insertion(+)

Index: polymake-4.13/include/core/polymake/internal/comparators.h
===================================================================
--- polymake-4.13.orig/include/core/polymake/internal/comparators.h
+++ polymake-4.13/include/core/polymake/internal/comparators.h
@@ -17,6 +17,7 @@
 
 #pragma once
 
+#include <cstdint>
 #include "polymake/internal/comparators_ops.h"
 #include "polymake/internal/matrix_rows_cols.h"
 #include "polymake/TransformedContainer.h"
